tools/diffscripts/RemoveHeadersFromDiffFile/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace RemoveHeadersFromDiffFile
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Take in the file name
+            string fileName = args[0];
+
+            // Get the existing contents
+            string fileContents = File.ReadAllText(fileName);
+
+            // Replace ```\r\n\r\n
+            string headerReplaceRegex = "^```\\s+^";
+            // Replace ## Microsoft...\r\n\r\n
+            string titleReplaceRegex = "^##.*\\s+^";
```

Comment:
Your regex doesn't match the comment. If you intend to replace (literally) `## Microsoft...\r\n\r\n`, why doesn't the regex have "Microsoft" in it?